hooks/boot/02-message-helpers.rb

```diff
@@ -6,7 +6,7 @@ def success_message
 
     def server_success_message(kafo)
       success_message
-      say "  * <%= color('Katello', :info) %> is running at <%= color('#{kafo.param('foreman', 'foreman_url').value}', :info) %>"
+      say "  * <%= color('Foreman', :info) %> is running at <%= color('#{kafo.param('foreman', 'foreman_url').value}', :info) %>"
```

Comment:
This changes the message from "Katello" to "Foreman" (while keeping "Katello" in the devel scenarios), which I think is fine, but might be unexpected by users? Should this be the scenario name instead?

Reply:
If it uses the scenario name, then it might also solve the downstream branding and remove the need for message helpers.

Reply:
The scenario name usage makes the assumption that thing running is always congruent with the name. I don't want to confuse users further with things like "Luna is running at" "Foreman Proxy with Content is running at". The software that's running in this case is Foreman (with plugins).

Reply:
Not for this PR, but we can think about storing that in the scenario. New fields can be added and I think I'd rather deal with that in scenarios than in ruby code.